.env.example

```diff
@@ -1,6 +1,8 @@
 # This is an example configuration for the stack.
 # Copy this to `.env` and adapt to your needs.
 
+BASE_URL=localhost
```

Comment:
I would not do that bug fix this issue instead: https://github.com/yverdon/geocity/issues/525

Reply:
Fixed by commit: https://github.com/nicolas-heigvd/geocity/commit/ed9dd0929e9dffc58bfb330a4522c300182d4174